plugins/MantisCoreFormatting/core/MantisMarkdown.php

```diff
@@ -205,57 +175,38 @@ protected function inlineUrl( $Excerpt ) {
 		return $this->processUrl( parent::inlineUrl( $Excerpt ) );
 	}
 
-	protected function inlineUrlTag( $Excerpt ) {
-		# @FIXME
-		# This function is supposed to process links like `<http://example.com>`
-		# on single-line texts, but it does not actually work: the function is
-		# never called (see Parsedown::line() 1077), because
-		# MantisCoreFormattingPlugin::formatted() applies html_specialchars()
-		# first, so the < > are converted to &lt;/&gt;.
+	protected function inlineUrlTag( $Excerpt )	{
 		return $this->processUrl( parent::inlineUrlTag( $Excerpt ) );
 	}
 
-
 	/**
-	 * Initialize the singleton static instance.
-	 */
-	private static function init() {
-		if ( null === static::$mantis_markdown ) {
-			static::$mantis_markdown = new MantisMarkdown();
-		}
-		return static::$mantis_markdown;
-	}
-
-	/**
-	 * Replace any '&amp;' entity in the given string by '&'.
-	 *
-	 * MantisBT text processing replaces '&' signs by their entity name. Within
-	 * code blocks or backticks, Parsedown applies the same transformation again,
-	 * so they ultimately become '&amp;amp;'. This reverts the initial conversion
-	 * so ampersands are displayed correctly.
+	 * Customize the blockHeader markdown
 	 *
-	 * @param string $p_text Text block to process
-	 * @return void
+	 * @param string $line The Markdown syntax to parse
+	 * @access protected
+	 * @return string|null HTML representation generated from markdown or null
```

Comment:
Should be `array[]|null` or even `array[]|void|null` since the parent Parsedown method calls `return` without a value.